src/catalog/CatalogPage.tsx

```diff
@@ -44,40 +47,67 @@ const CatalogPage = () => {
   const totalCourses = courseData?.results?.length ?? 0;
 
   return (
-    <Container className="container-xl pt-5.5">
-      <SubHeader title={intl.formatMessage(messages.totalCoursesHeading, {
-        totalCourses,
-      })}
+    <Container className="container-xl pt-5.5 mb-6">
+      <SubHeader
+        title={intl.formatMessage(messages.exploreCourses)}
+        className={classNames({ 'mx-2.5': isMedium })}
```

Comment:
Kind of frustrating to need to add this to match the `DataTable`, but I can't think of a better solution.

[curious] the `DataTable` seems to be using padding instead of margins, is there a specific reason to use `mx-2.5` instead of `px-2.5` here? I don't have strong feelings either way.

Reply:
I don’t have strong feelings about this either, but it seems to me that using margins is preferable for external spacing. Perhaps this excerpt from the article [Margin vs Padding (Difference and Usage)](https://eu.siteground.com/kb/margin-vs-padding/) supports this perspective.

<img width="646" height="201" alt="image" src="https://github.com/user-attachments/assets/7db4f1d6-3f63-4231-a409-9486eed4c127" />